ui/tests/integration/components/dashboard/replication-state-text.js

```diff
@@ -24,13 +24,12 @@ module('Integration | Component | dashboard/replication-state-text', function (h
 
   test('it displays replication states', async function (assert) {
     await render(
-      hbs`
-        <Dashboard::ReplicationStateText 
-          @name={{this.name}} 
-          @version={{this.version}} 
-          @subText={{this.subText}} 
-          @clusterStates={{this.clusterStates}} />
-          `
+      hbs`<Dashboard::ReplicationStateText
```

Comment:
❓not related to the open API, what linter do you use on vscode? When I created these tests, my linter didn't seem to format things correctly ☹️

Reply:
Prettier - Code formatter is the default I have set for test files. But this change only happens when I use the `ember g` command so not sure exactly what that's using